routes/users.js

```diff
@@ -0,0 +1,128 @@
+const express = require("express");
+const usersRouter = express.Router();
+
+const jwt = require("jsonwebtoken");
+const { JWT_SECRET } = process.env;
+
+const bcrypt = require("bcrypt");
+
+const { getUserByUsername, createUser } = require("../db");
+
+usersRouter.post("/register", async (req, res, next) => {
+  const {
+    username,
+    password,
+    firstName,
+    lastName,
+    email,
+    imageURL,
+    isAdmin,
+  } = req.body;
+
+  try {
+    const _user = await getUserByUsername(username);
+
+    if (_user) {
+      next({
+        name: "UserExistsError",
+        message: "A user by that username already exists",
+      });
+      return;
+    }
+    if (password.length < 8) {
+      next({
+        name: "ShortPassword",
+        message: "Password Too Short!",
+      });
+      return;
+    }
+    const user = await createUser({
+      username: username,
+      password: password,
+      firstName: firstName,
+      lastName: lastName,
+      email: email,
+      imageURL: imageURL,
+      isAdmin: isAdmin,
```

Comment:
We are allowing anyone with access to postman to create an admin user.  Try doing something like 
`if(req.user.isAdmin) toInsert.isAdmin = req.body.isAdmin;`

Reply:
DEFINITELY take care of this one!